query-node/mappings/src/content/app.ts

```diff
@@ -49,32 +52,19 @@ export async function processCreateAppMessage(
     platforms: appMetadata?.platforms || undefined,
     category: appMetadata?.category || undefined,
     authKey: appMetadata?.authKey || undefined,
-    ownerCuratorGroup: channel.ownerCuratorGroup,
-    ownerMember: channel.ownerMember,
-    channel,
   })
   await store.save<App>(newApp)
   logger.info('App has been created', { name })
 }
 
-async function createAppId(event: SubstrateEvent): Promise<string> {
-  return `${event.blockNumber}-${event.indexInBlock}`
-}
-
-export async function processUpdateApp(
-  store: DatabaseManager,
-  channelId: ChannelId,
-  message: IUpdateApp
-): Promise<void> {
+export async function processUpdateAppMessage(store: DatabaseManager, message: IUpdateApp): Promise<void> {
   const { appId, appMetadata } = message
 
-  const app = await getAppById(store, appId)
+  const app = await getAppByIdAndMemberId(store, appId)
```

Comment:
The function is called `getAppByIdAndMemberId` but only the `appId` is used in the query.
There is actually a security issue here too, because you're not checking anywhere that `app.ownerMember` is actually the member that submitted the remark transaction (or lead in case `app.ownerMember` is not set)